backend/pkg/plugins/plugins.go

```diff
@@ -103,6 +113,31 @@ func GeneratePluginPaths(staticPluginDir string, pluginDir string) ([]string, er
 	return pluginListURL, nil
 }
 
+var GenerateSeparatePluginPathsFunc = GenerateSeparatePluginPaths
+
+// ListPlugins lists the plugins in the static and user-added plugin directories.
+func ListPlugins(staticPluginDir, pluginDir string) error {
+	staticPlugins, userPlugins, err := GenerateSeparatePluginPathsFunc(staticPluginDir, pluginDir)
+	if err != nil {
+		logger.Log(logger.LevelError, nil, err, "listing plugins")
+		return fmt.Errorf("error listing plugins: %w", err)
+	}
+
+	fmt.Println("  Static Plugins:")
```

Comment:
Can you please use `logger.Log` with Info field on this?

Reply:
I think using Print would be a lot easier to read in general as parsing the logs for the list of plugins seems tedious, WDYT?

Reply:
Hmm, as you mentioned earlier we just need to list plugins and since we are not running a server here. Folks won’t need to parse the logs so I think fmt can work fine for this case. Cool